Makefile

```diff
@@ -16,10 +11,8 @@ build-web: setup build-lib
 
 .PHONY: setup
 setup: bun.lock
```

Comment:
```suggestion
setup:
```

Reply:
Good catch. Fixed. Also removed a reference to `bun.lock` in our `make clean`.